disnake/utils.py

```diff
@@ -595,6 +595,11 @@ def compute_timedelta(dt: datetime.datetime):
     return max((dt - now).total_seconds(), 0)
 
 
+def validate_guild_ids(guild_ids: Iterable[int]) -> None:
+    if not all(isinstance(guild_id, int) for guild_id in guild_ids):
+        raise ValueError("test_guilds or guild_ids must be a sequence of int.")
```

Comment:
I don't think that we need a public function in `utils.py` for that purpose, it seems like something internal.
I suggest removing it and using `if not all(isinstance(guild_id, int) for guild_id in guild_ids)`